personas.md

```diff
@@ -0,0 +1,38 @@
+# OSAC Personas
+
+These are the essential personas that OSAC must target for an MVP. Many
+additional personas can be identified within the solution space, some of which
+will be added to this document over time as the project is ready to address
+their unique use cases.
+
+## Cloud Provider Admin
+
+* is an associate of the Cloud Provider
```

Comment:
can we change " associate of" to works for?  Associate is not a globally understood term.